doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -106,6 +106,8 @@ Matter
 * Disabled OpenThread shell by default in Matter over Thread samples.
 * Enabled :kconfig:option:`CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option by default, including for builds without factory data support.
   The firmware now erases all flash pages in the non-volatile storage during a factory reset, instead of just clearing Matter-related settings.
+* Added shell commands for printing and resetting the top usage or critical resources used by the Matter stack.
```

Comment:
```suggestion
* Added shell commands for printing and resetting the top usage of critical resources used by the Matter stack.
```

Reply:
What is `top usage` here?

Reply:
maximum usage

Reply:
Could we say "maximum usage"?

Reply:
Yes, "top" usage is wrong here, but "peak usage" is something commonly used in this context. How about that?